oktapam/resource_group.go

```diff
@@ -156,3 +159,12 @@ func resourceGroupDelete(ctx context.Context, d *schema.ResourceData, m interfac
 	d.SetId("")
 	return diags
 }
+
+func importResourceGroupState(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	// d.Id() here is the last argument passed to the `terraform import RESOURCE_TYPE.RESOURCE_NAME RESOURCE_ID` command
+	// Set the passed id as group name to make read work
+	if err := d.Set(attributes.Name, d.Id()); err != nil {
```

Comment:
Looking at [L98](https://github.com/okta/terraform-provider-oktapam/pull/69/files#diff-026ab8497d618ddb80d70b8a9337e757751db813cf73b5183f0bc0343f42f21aR98) it looks like the ID is being set to the UUID, not name.

Reply:
Last argument in the command is passed as *schema.ResourceData Id (d.Id()) to import handler function. Some of the resources cannot be fetched using ASA resource UUID. So, all the required attributes must be passed to import command. Importer parse the input and set the required attributes in the state for read to work. Read context function assumes that required attributes are already set in the state.
      importer handler function calls -->  read context
      
Project cannot be fetched using Project UUID.